### ISSUE TITLE
[Feature Request]Choosable Mats/Info display

### ISSUE THREAD

**Issue by kenahamtan** [KC3Kai/KC3Kai#2275]
Some version ago, mats like screw, blow torch, current equipment#/max equipment#, current # of fleet girl/max # of fleet girl were displayed.  I like that.  I don't have to go into profile game page to view these info.
Then these infos were replaced by resources which very obvious can be graps from the base page.  I guess some ttk may want it so.
Could resources(4 types), screws, buckets, blow torch, build mats, current # of equipment, current # of fleet girls be made as selectable options (with a max of 6 display options) and let ttk choose whichever we like to be displayed in the panel?

**Comment by MegaAbsol** [comment]
You can click on the view to change it. It cycles through your resources, your devmats/blowtorches/FCM and stuff, and your  current equipment#/max equipment#, current # of fleet girl/max # of fleet girl stuff.

TL;DR just click where you see your resources to cycle through them.
